tests/unit/amo/components/TestAddon.js

```diff
@@ -44,7 +44,10 @@ import {
   dispatchSignInActions, fakeAddon, signedInApiState,
 } from 'tests/unit/amo/helpers';
 import {
-  createFetchAddonResult, getFakeI18nInst, sampleUserAgentParsed,
+  createFetchAddonResult,
+  getFakeI18nInst,
+  sampleUserAgentParsed,
+  createStubErrorHandler,
```

Comment:
Total nitpick, but: could these be alphabetised please? I wish we had an ESLint rule for this but I dunno if one exists. Our convention is to alphabetise imports though.